pkg/controller/authentication/authentication_controller.go

```diff
@@ -171,6 +176,26 @@ func (r *ReconcileAuthentication) Reconcile(ctx context.Context, request reconci
 		return reconcile.Result{}, err
 	}
 	r.status.OnCRFound()
+
+	//Set the meta info in the tigerastatus like observedGenerations
```

Comment:
You corrected others, but no this one. Please end with a period and add a space after "//"

```
// Set metadata in the TigeraStatus such as observedGenerations.
```